### PR TITLE
Start a toolkit for Kubernetes

### PR DESCRIPTION
**Pull request by johnmuth81**
First tool is a node file system examination tool.

### CODE REVIEW

**Review by sv641 — APPROVED** [review]

File: cloud/k8s/tools/hack_node.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kontain-node-hack
+  namespace: kube-system
```
Comment (smijar) [inline cloud/k8s/tools/hack_node.yaml:21]: Maybe use a different namespace for this like 'default' else the developer might forget to clean up the pod in kube-system thus affecting the quota for the namespace.

Comment (smijar) [reply cloud/k8s/tools/hack_node.yaml:21]: If you leave namespace blank it defaults to the current context.

File: cloud/k8s/tools/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+This creates a privileged DaemonSet in the `kube-system` namespace with a base name of `kontain-node-hack`
+to run on all Nodes. The daemon set mounts the Node's root file system `/` as `/host-root` and sleeps forever.
+
+The pod(s) running the DamonSet are found with `kubectl get pod -A -o wide`. A shell can them be started
+on a DamonSet pod with `kubectl exec -it`.
+
+### Tips and Tricks
+
+- Once logged into the DaemonSet pod, the Nodes `/proc` is mounted as `/host-root/proc`, so the command `chroot /host-root ps -aef` will show all processes running on the Node.
```
Comment (smijar) [inline cloud/k8s/tools/README.md:15]: Also found out that if the user uses $ chroot /host-root, then he can use the host's normal processes like a regular user.

Comment (johnmuth81) [reply cloud/k8s/tools/README.md:15]: How so? Did `kill` work? (that would surprise me).

**Pull request merged**